src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/JiraCoreScenarioDevloop.kt

```diff
@@ -0,0 +1,54 @@
+package com.atlassian.performance.tools.jiraactions.api.scenario
+
+import com.atlassian.performance.tools.dockerinfrastructure.api.jira.Jira
+import org.junit.Ignore
+import org.junit.Test
+import org.openqa.selenium.PageLoadStrategy
+import org.openqa.selenium.chrome.ChromeDriver
+import org.openqa.selenium.chrome.ChromeOptions
+import org.openqa.selenium.logging.LogType
+import org.openqa.selenium.logging.LoggingPreferences
+import org.openqa.selenium.remote.CapabilityType
+import org.openqa.selenium.remote.RemoteWebDriver
+import java.net.URI
+import java.util.logging.Level
+
+/**
+ * This class is meant to be used in a local devloop, where JIra is already running.and Chrome is locally available.
```

Comment:
Oooh, I get it. We can have tests that work locally and not on CI.
Not having to provision Jira every time is gonna be faster for sure.
OTOH the dev still has to provision manually Jira at least once, so the first loop is still slow (and tribal). We need Jira to start quickly. What's the duration of `JiraCoreFormula.provision()`?

Reply:
I have data for the full run - it's 12 minutes vs 1m30s . Obviously, if you need to test just one action, it becomes 12minutes vs 10-20 seconds...

Reply:
That's convincing!

Reply:
Do you need this for ShipIt? I'd like to review some small stuff here, but I'm participating in the ShipIt right now, so I'm deferring PRs by default

Reply:
No, keep shipping!

Reply:
As this test is intended for manual execution only it won't get executed on CI which makes it prone to degradation in the future. That said, the gain for a dev is so big that it indeed might make sense to have it.